src/Infrastructure/Rsp.IrasService.Infrastructure/Repositories/ReviewBodyRepository.cs

```diff
@@ -56,6 +57,21 @@ public async Task<ReviewBody> UpdateReviewBody(ReviewBody reviewBody)
 
         if (reviewBodyEntity == null) return reviewBodyEntity;
         reviewBodyEntity.IsActive = false;
+        reviewBodyEntity.UpdatedDate = DateTime.Now;
+        await irasContext.SaveChangesAsync();
+
+        return reviewBodyEntity;
+    }
+
+    public async Task<ReviewBody?> EnableReviewBody(Guid id)
+    {
+        var reviewBodyEntity = await irasContext
+            .ReviewBodies
+            .SingleOrDefaultAsync(r => r.Id == id);
+
+        if (reviewBodyEntity == null) return reviewBodyEntity;
```

Comment:
Please avoid using single line if statements.

```suggestion
        if (reviewBodyEntity == null)
        {
             return reviewBodyEntity;
        }
```